previews/azure-functions-service-bus.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Azure Functions with Azure Service Bus
+component: ASBFunctions
+summary: Azure Functions Preview
```

Comment:
```suggestion
summary: Hosting NServiceBus endpoints with Azure Functions and Azure Service Bus triggers.
```

Reply:
> Hosting NServiceBus endpoints with Azure Functions and Azure Service Bus triggers.

When I read this I think of "hosting NServiceBus endpoints using technology X and Y" where in reality the hosting is done with X and Y is different than hosting aspect. What about the following?

> Hosting NServiceBus endpoints with Service Bus triggered Azure Functions.

Reply:
I'll update both documents with the summary I've proposed. In case you think that's not enough, let's issue a follow-up PR for docs and clean it up. Thank you.